flink-tests/src/test/java/org/apache/flink/test/runtime/FileBufferReaderITCase.java

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
I think it would be better to squash the ITCase commit with the bug fix. Whenever I add a test coverage for some bug, I try to keep it together, as the test might provide some extra context for whoever will be reading the bug fix commit in the future, like how the bug was triggered before the bug fix, what were the conditions/setup, etc.

Reply:
Yes, I agree with squashing this ITCase commit with core commit when merging. Make it separate now is used for verifying this fix work.